x/slashing/simulation/operations_test.go

```diff
@@ -105,38 +106,33 @@ func (suite *SimTestSuite) TestWeightedOperations(t *testing.T) {
 
 	weightesOps := simulation.WeightedOperations(appParams, cdc, suite.accountKeeper, suite.bankKeeper, suite.slashingKeeper, suite.stakingKeeper)
 	for i, w := range weightesOps {
-		operationMsg, _, err := w.Op()(r, app.BaseApp, ctx, accs, ctx.ChainID())
-		require.NoError(t, err)
+		operationMsg, _, err := w.Op()(suite.r, app.BaseApp, ctx, accs, ctx.ChainID())
+		suite.Require().NoError(err)
 
 		// the following checks are very much dependent from the ordering of the output given
 		// by WeightedOperations. if the ordering in WeightedOperations changes some tests
 		// will fail
-		suite.Require().Equal(t, expected[i].weight, w.Weight(), "weight should be the same")
-		suite.Require().Equal(t, expected[i].opMsgRoute, operationMsg.Route, "route should be the same")
-		suite.Require().Equal(t, expected[i].opMsgName, operationMsg.Name, "operation Msg name should be the same")
+		suite.Require().Equal(expected[i].weight, w.Weight(), "weight should be the same")
+		suite.Require().Equal(expected[i].opMsgRoute, operationMsg.Route, "route should be the same")
+		suite.Require().Equal(expected[i].opMsgName, operationMsg.Name, "operation Msg name should be the same")
 	}
 }
 
 // TestSimulateMsgUnjail tests the normal scenario of a valid message of type types.MsgUnjail.
 // Abonormal scenarios, where the message is created by an errors, are not tested here.
-func (suite *SimTestSuite) TestSimulateMsgUnjail(t *testing.T) {
-	// setup 3 accounts
-	s := rand.NewSource(5)
-	r := rand.New(s)
+func (suite *SimTestSuite) TestSimulateMsgUnjail() {
 	app, ctx, accounts := suite.app, suite.ctx, suite.accs
+
 	blockTime := time.Now().UTC()
 	ctx = ctx.WithBlockTime(blockTime)
 
-	// remove genesis validator account
-	accounts = accounts[1:]
```

Comment:
The genesis validator is never in the accounts, so we should not need to remove it.